riscv/processor.cc

```diff
@@ -77,6 +77,13 @@ processor_t::~processor_t()
     fprintf(stderr, "PC Histogram size:%zu\n", pc_histogram.size());
     for (auto it : pc_histogram)
       fprintf(stderr, "%0" PRIx64 " %" PRIu64 "\n", it.first, it.second);
+
+    fprintf(stderr, "Instruction Name Histogram size:%zu\n", insn_name_histogram.size());
+    for (auto it : insn_name_histogram) {
+      auto insn_name = new char[it.first.size() + 1] { '\0' };
```

Comment:
memory leak?

Reply:
I see, C++ requires explicit deconstruct. Thank you, let me update it.

Reply:
Updated.